Apps/AdminWebClient/src/ClientApp/src/views/Feedback.vue

```diff
@@ -136,10 +143,14 @@ export default class FeedbackView extends Vue {
 
     private userFeedbackService!: IUserFeedbackService;
 
-    private focusedTags: AdminTag[] = [];
+    private focusedTags: UserFeedbackTag[] = [];
 
     private isLoadingTag = false;
 
+    private get availiableTags(): string[] {
```

Comment:
Typo: `availiable` should be `available`.

Reply:
its fixed